internal/version/version.go

```diff
@@ -5,20 +5,96 @@ package version
 
 import (
 	"encoding/json"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
 )
 
-// V contains all necessary values for comparing different binary version with each other.
-// It's goal is to limit the startup up outdated binaries as well as having an easier way to ensure the software
-// is on the latest version from a UI/UX perspective
+var (
+	// Timestamp is the UTC timestamp of the compilation time
+	Timestamp string
+	// CommitHash is the git hash of the code being compiled
+	CommitHash string
+	// Version is the semantic version set at compilation
+	// if not a valid semantic version Release should be false
+	Version = "v0.0.1"
```

Comment:
i think it's okay to leave this empty if go-release wasn't used

Reply:
Its more a temporary solution to run the tools 👍
will be reset to `Version string` once its working

Reply:
okay i guess i should say - i think i would prefer non-release binaries to not have a version at all

Reply:
Yeah thats what i wanted to point out with that. Currently i set it as default to let the binaries start